### PR TITLE
Add links to Google Docs for each projects.

### PR DESCRIPTION
**Pull request by Mariatta**

### CODE REVIEW

File: docs/communication.rst [hunk 1]
```diff
@@ -29,3 +29,7 @@ share the links.
 
 When you're in a group meeting, please remember to take down notes, so those
 unable to attend the meeting can get up speed.
+
+We have created a Google Doc for each projects. You can find the link to
```
Comment (freddrake) [inline docs/communication.rst:33]: "each project"

File: docs/communication.rst [hunk 2]
```diff
@@ -29,3 +29,7 @@ share the links.
...
+the notes in :ref:`projects`. We will ensure that each participants will
```
Comment (freddrake) [inline docs/communication.rst:34]: "each participant"

File: docs/projects.rst [hunk 3]
```diff
@@ -8,6 +8,8 @@ Sprint Projects
 
    :ref:`participants` for the list of all sprint participants.
 
+As it gets closer to the sprint, we will ensure that all participants will have
+access to each notes linked below.
```
Comment (freddrake) [inline docs/projects.rst:12]: "each note" or "appropriate notes documents", whichever is accurate.

**Review by freddrake — CHANGES_REQUESTED** [review]
Just a few wording nits.

File: docs/projects.rst (hunk 3 above)
Comment (Mariatta) [reply docs/projects.rst]: Thanks, I've changed it to "all notes".

**Review by freddrake — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**